webapp/public/index.html

```
@@ -0,0 +1,402 @@
+  <!-- ─── Report slide-in panel ───────────────────────────────── -->
+  <div class="modal-overlay" id="modal-overlay" role="dialog"
+       aria-modal="true" aria-labelledby="report-company">
+    <div class="report-panel" id="report-panel" tabindex="-1">
+
+      <div class="report-panel-header">
+        <div class="report-panel-info">
+          <div class="report-panel-company" id="report-company">Loading…</div>
+          <div class="report-panel-role"   id="report-role"></div>
+          <div class="report-panel-meta"   id="report-meta"></div>
+        </div>
+        <button class="close-btn" id="close-report" aria-label="Close report">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)